luxtronik/__init__.py

```diff
@@ -233,3 +266,49 @@ def _read_char(self):
         "Low-level helper to receive a signed int"
         reading = self._socket.recv(LUXTRONIK_SOCKET_READ_SIZE_CHAR)
         return struct.unpack(">b", reading)[0]
+
+
+class Luxtronik(LuxtronikData):
+    """
+    Wrapper around the data and the read/write interface.
+    Mainly to ensure backwards compatibility
+    of the read/write interface to other projects.
+    """
+
+    def __init__(self, host, port=LUXTRONIK_DEFAULT_PORT, safe=True):
+        super().__init__(Parameters(safe), Calculations(), Visibilities())
```

Comment:
With my proposed constructor of `LuxtronikData`, this could be just `super().__init__(safe=safe)`.

Reply:
Done